security/password.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Automattic\VIP\Security;
+
+use WP_Error;
+use WP_User;
+
+add_action( 'user_profile_update_errors', __NAMESPACE__ . '\validate_current_password', 1, 3 );
+
+/**
+ * Validate current password in submitted user profile.
+ *
+ * @param WP_Error $errors Error object passed by reference
+ * @param bool $update Whether this is a user update
+ * @param WP_User $user User object (passed by reference)
+ *
+ * @return void
+ */
+function validate_current_password( WP_Error $errors, bool $update, WP_User $user ) {
```

Comment:
I initially tried `stdClass` as per the documentation, but as you point out `object` would be a better fit here since it satisfies `WP_User` (which is what I it's being passed in multiple places).

Reply:
Regarding passing by reference, I removed it because this IDE suggestion:

<img width="625" alt="Screen Shot 2021-04-12 at 9 26 54 AM" src="https://user-images.githubusercontent.com/7188409/114356781-48cdf900-9b71-11eb-8bcd-c4b364f6b394.png">